index.js

```diff
@@ -10,44 +10,44 @@ module.exports.default = parse
  * conversion ratios
  */
 
-parse.nanosecond =
-parse.ns = 1 / 1e6
+parse.year =
+  parse.yr =
+  parse.y = 60000 * 60 * 24 * 365.25
 
-parse['µs'] =
-parse['μs'] =
-parse.us =
-parse.microsecond = 1 / 1e3
+parse.month =
+  parse.b = 60000 * 60 * 24 * (365.25 / 12)
 
-parse.millisecond =
-parse.ms =
-parse[''] = 1
+parse.week =
+  parse.wk =
+  parse.w = 60000 * 60 * 24 * 7
 
-parse.second =
-parse.sec =
-parse.s = parse.ms * 1000
+parse.day =
+  parse.d = 60000 * 60 * 24
+
+parse.hour =
+  parse.hr =
+  parse.h = 60000 * 60
 
 parse.minute =
-parse.min =
-parse.m = parse.s * 60
+  parse.min =
+  parse.m = 60000
 
-parse.hour =
-parse.hr =
-parse.h = parse.m * 60
+parse.second =
+  parse.sec =
+  parse.s = 1000
 
-parse.day =
-parse.d = parse.h * 24
+parse.millisecond =
+  parse.millisec =
+  parse.ms = 1
 
-parse.week =
-parse.wk =
-parse.w = parse.d * 7
+parse['µs'] =
+  parse['μs'] =
+  parse.us =
+  parse.microsecond = 1 / 1e3
 
-parse.month =
-parse.b =
-parse.d * (365.25 / 12)
+parse.nanosecond =
+  parse.ns = 1 / 1e6
 
-parse.year =
-parse.yr =
-parse.y = parse.d * 365.25
 
```

Comment:
```js
parse.custom = 100000
return parse("1h 1")
```
That returns 3660000 (one hour and one minute), ignoring the new time unit (which can't be added in order).
We can hope that this won't be an issue, the example in the README file only shows adding an alias to existing time units.

Reply:
Well there is a way to define custom units in right order, but that would take deleting/readding all units.

Reply:
A fix for it would be creating the `order` variable (like in my version) when using the `parse` function for the first time, so between when the module is loaded and when it is first used (which should be when developers define custom units).

Reply:
You are right. But looking at it with pragmatic perspective makes it an overkill. I guess common use is either extending (l10n) or silent units (your case). Mixing these two makes me wonder if person should even use this lib to parse such exotics.
Anyways thanks for findings!